docs/framework/interop/default-marshaling-behavior.md

```diff
@@ -215,7 +215,7 @@ public struct Rect {
 }  
 ```  
   
- When marshaled to unmanaged code, these formatted types are marshaled as C-style structures. This provides an easy way of calling an unmanaged API that has structure arguments. For example, the `POINT` and `RECT` structures can be passed to the Microsoft Win32 API **PtInRect** function as follows:  
+ When marshaled to unmanaged code, these formatted types are marshaled as C-style structures. This provides an easy way of calling an unmanaged API that has structure arguments. For example, the `POINT` and `RECT` structures can be passed to the Microsoft Windows API **PtInRect** function as follows:  
   
```

Comment:
- nit unrelated to your PR: line 220 should have a language identifier (csharp)
- The occurrences of Win32API should also be changed to WindowsAPI in code.

Reply:
No, that's in C++.